### PR TITLE
Edit "Best of" and add "Community Choice" and crowns

### LINKED ISSUE #982: Include Community Choice (formerly Best of) in the Contests article
(Note: the description of this issue was modified to match up with the most recent comments)

Best of and Community Choice articles should be removed. Replacing it will be links in the Contest page.

### LINKED ISSUE #1101: No crowns on "Best of series"
Saw the crown icons on top of code. Also when on web, pressing those numbers will go to the icon. I'll do it if it was intended.

### LINKED ISSUE #1115: "Contest" ranking system should be the same on "best of/community choice" pages
Edit: This might help- https://en.wikipedia.org/wiki/Ranking
Also on:
ppy/osu-web#2637 (for contest page)
On the wiki, the pages will change from all beatmaps to only top 50 from #1105, but wondering if the ranks should be the same, In the contests, it only shows that one beatmap and not the other on the spreadsheet. On the wiki, I put that other beatmap b/c it was upper. A whole new problem is when there are multiple beatmaps with the same pts(votes) on the same listing.

Edit: This is dense ranking, after some research.
~Plan to have white space. Now 2009 should have 28 placings. Here there are 2 beatmaps with the same vote.~ Pick a ranking system.
Placing | Votes
 
~Edit: This, duplicate numbers and crowns, or multiple beatmaps on the same row.~ Idk pick a ranking system.
~Edit: I don't know how placing actually works.~ I did research.

### PR DESCRIPTION
**Pull request by Joehuu**
### Description

Reworded "main page" to "part of" series in redirect links. Added Community Choice and crowns to 1st, 2nd, and 3rd place. Add redirects to best of 2017 and community choice 2016 and below. Add links to go to their other counterpart in their pages.
Edit: Only did first 50 on the 2017 page. I think adding all of it is too much.
Edit: Will change the ranks, so top 50 will have more than 50 beatmaps.

### Status

- [X] WIP

### Additional Notes 

Resolves #982 and resolves #1101.
Edit: Will resolve #1115.

### CODE REVIEW

File: wiki/Best_Of/2010/en.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
+[#1]: /wiki/shared/GCrown.png "1st"
+[#2]: /wiki/shared/SCrown.png "2nd"
+[#3]: /wiki/shared/BCrown.png "3rd"
```
Comment (TPGPL) [inline wiki/Best_Of/2010/en.md]: add ` place` to it in the articles

File: wiki/Best_Of/2010/en.md [hunk 2]
```diff
@@ -9,55 +15,61 @@ _Main page: [Best Of](/wiki/Best_Of "Best Of")_
   - [Part 1](https://www.mediafire.com/?0hekhu37h5bbeok)
   - [Part 2](https://www.mediafire.com/?rfpzf1c1detavfz)
 
-|Placing|Votes|Beatmap|
+## Modes
+
+_Only osu!standard for this year._
```
Comment (TPGPL) [inline wiki/Best_Of/2010/en.md]: for this year's edition?

Comment (Joehuu) [reply wiki/Best_Of/2010/en.md]: Yes. Should it be reworded?

Comment (TPGPL) [reply wiki/Best_Of/2010/en.md]: prob

**Comment by Joehuu** [comment]
Gonna redo this.

**Pull request closed without merging**